src/classes/RD2_Settings.cls

```diff
@@ -121,6 +121,24 @@ public with sharing class RD2_Settings {
         private set;
     }
 
+    /***
+    * @description Installment Opp Auto Creation Option,
+    * If random string is input, the setting will be default to Always Create Next installment
+    */
+    @TestVisible
+    public String firstInstallmentAsync {
```

Comment:
`firstInstallmentOppCreateOption` or `installmentOppCreateFirstOption` since async is one of the modes.

Reply:
What do you think of `firstInstallmentCreateMode`?